pluggy/manager.py

```diff
@@ -251,16 +251,22 @@ def check_pending(self):
                                 % (name, hookimpl.plugin),
                             )
 
-    def load_setuptools_entrypoints(self, entrypoint_name):
-        """ Load modules from querying the specified setuptools entrypoint name.
-        Return the number of loaded plugins. """
+    def load_setuptools_entrypoints(self, group, name=None):
+        """ Load modules from querying the specified setuptools ``group``.
+
+        :param str group: entry point group to load plugins
+        :param str name: if given, loads only plugins with the given ``name``.
+        :rtype: int
+        :return: return the number of loaded plugins by this call.
+        """
         from pkg_resources import (
             iter_entry_points,
             DistributionNotFound,
             VersionConflict,
         )
 
-        for ep in iter_entry_points(entrypoint_name):
+        count = 0
+        for ep in iter_entry_points(group, name=name):
```

Comment:
`for count, ep in enumerate(iter_entry_points(group, name=name)):` maybe a little cleaner?

Reply:
it would be wrong ^^ - see the continue cases

Reply:
a good call. missed that cause of the stacked diff :confounded: